dashboard/index.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+require_once("DataProcessor.php");
+
+$containerDataProcessor = new DataProcessor();
+$data = $containerDataProcessor->process();
+
+$tbodyData = '';
+foreach ($data as $dataEntry) {
+    if (count($dataEntry['domains']) < 1) {
+        $tbodyData .= '<tr><td>' . $dataEntry['name'] . '</td><td>' . $dataEntry['domains'][0] . '</td></tr>';
+    }
+    else {
+        foreach ($dataEntry['domains'] as $index => $domain) {
+            if ($index === 0) {

```

Comment:
that looks too bulky for what it actually does

Reply:
If there are multiple domains present, the name of the container should be shown only for the first row. I have yet to find a leaner way to write this function